backend/de.metas.acct.base/src/main/sql/postgresql/system/43-de.metas.acct/5671780_sys_gh14367_create_report_processes.sql

```diff
@@ -47,7 +47,7 @@ UPDATE AD_Process SET SQLStatement='SELECT * FROM de_metas_acct.report_bank_in_t
 -- Value: report_inventory_clearing
 -- Classname: de.metas.impexp.spreadsheet.process.ExportToSpreadsheetProcess
 -- 2023-01-13T16:02:30.140Z
-INSERT INTO AD_Process (AccessLevel,AD_Client_ID,AD_Org_ID,AD_Process_ID,AllowProcessReRun,Classname,CopyFromProcess,Created,CreatedBy,EntityType,IsActive,IsApplySecuritySettings,IsBetaFunctionality,IsDirectPrint,IsFormatExcelFile,IsLogWarning,IsNotifyUserAfterExecution,IsOneInstanceOnly,IsReport,IsTranslateExcelHeaders,IsUseBPartnerLanguage,LockWaitTimeout,Name,PostgrestResponseFormat,RefreshAllAfterExecution,ShowHelp,SpreadsheetFormat,SQLStatement,Type,Updated,UpdatedBy,Value) VALUES ('3',0,0,585180,'Y','de.metas.impexp.spreadsheet.process.ExportToSpreadsheetProcess','N',TO_TIMESTAMP('2023-01-13 18:02:29','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.acct','Y','N','N','N','Y','N','N','N','N','Y','Y',0,'In Rechnung gestellte Ware','json','N','N','xls','SELECT * FROM de_metas_acct.report_inventory_clearing(@C_ElementValue_ID/null@, @C_AcctSchema_ID/null@, @EndDate/null@);','Excel',TO_TIMESTAMP('2023-01-13 18:02:29','YYYY-MM-DD HH24:MI:SS'),100,'report_inventory_clearing')
+INSERT INTO AD_Process (AccessLevel,AD_Client_ID,AD_Org_ID,AD_Process_ID,AllowProcessReRun,Classname,CopyFromProcess,Created,CreatedBy,EntityType,IsActive,IsApplySecuritySettings,IsBetaFunctionality,IsDirectPrint,IsFormatExcelFile,IsLogWarning,IsNotifyUserAfterExecution,IsOneInstanceOnly,IsReport,IsTranslateExcelHeaders,IsUseBPartnerLanguage,LockWaitTimeout,Name,PostgrestResponseFormat,RefreshAllAfterExecution,ShowHelp,SpreadsheetFormat,SQLStatement,Type,Updated,UpdatedBy,Value) VALUES ('3',0,0,585180,'Y','de.metas.impexp.spreadsheet.process.ExportToSpreadsheetProcess','N',TO_TIMESTAMP('2023-01-13 18:02:29','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.acct','Y','N','N','N','Y','N','N','N','N','Y','Y',0,'In Rechnung gestellte Ware','json','N','N','xls','SELECT * FROM de_metas_acct.report_inventory_clearing(@C_ElementValue_ID/null@, @C_AcctSchema_ID/null@, ''@EndDate/null@'');','Excel',TO_TIMESTAMP('2023-01-13 18:02:29','YYYY-MM-DD HH24:MI:SS'),100,'report_inventory_clearing')
```

Comment:
@adi-stefan  not sure if `''@EndDate/null@''` would work because in case of date not set it will send the string `'null'` to function which will lead to errors.

i suggest to make the EndDate parameter mandatory and set the default value to `@#Date@`. Would do the job.

Reply:
right